update.sh

```diff
@@ -160,6 +160,7 @@ function update_node_version() {
         curl -sSL --compressed "https://unofficial-builds.nodejs.org/download/release/v${nodeVersion}/SHASUMS256.txt" | grep "node-v${nodeVersion}-linux-x64-musl.tar.xz" | cut -d' ' -f1
       )
       if [ -z "$checksum" ]; then
+        rm -f "${dockerfile}-tmp"
```

Comment:
should we do some `git checkout -f` as well?

Reply:
In this case, a simple removal should be good enough?